addOns/graphql/graphql.gradle.kts

```diff
@@ -32,8 +32,13 @@ zapAddOn {
 
 dependencies {
     compileOnly(parent!!.childProjects.get("automation")!!)
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.6")
     implementation("com.google.code.gson:gson:2.8.6")
     implementation("com.graphql-java:graphql-java:15.0")
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.6")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.12.0")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.12.0")
+    implementation("org.snakeyaml:snakeyaml-engine:2.2.1")
```

Comment:
It's preferable to make these `api` dependencies in the automation add-on, otherwise this leads to duplication in each add-on.